decidim-core/app/packs/src/decidim/index.js

```diff
@@ -70,7 +70,7 @@ import markAsReadNotifications from "src/decidim/notifications"
 import handleNotificationActions from "src/decidim/notifications_actions"
 import RemoteModal from "src/decidim/remote_modal"
 import createTooltip from "src/decidim/tooltips"
-import fetchRemoteTooltip from "src/decidim/remote_tooltips"
+// import fetchRemoteTooltip from "src/decidim/remote_tooltips"
```

Comment:
```suggestion
// Temporary disabling this feature because we have a poor performance. See https://github.com/decidim/decidim/issues/14431 
// import fetchRemoteTooltip from "src/decidim/remote_tooltips"
```